NEWS.md

```diff
@@ -2,6 +2,8 @@
 
 * include tests for cloneWorksheet
 
+* Fixed NEWS hyperlink
+
 # openxlsx 4.1.4
 
 * Use `zip::zipr()` instead of zip::zip().
```

Comment:
* Use `zip::zipr()` instead of `zip::zip()`.

Could you also correct this line?